packages/web/src/pages/track-page/components/desktop/TrackPage.tsx

```diff
@@ -183,7 +172,6 @@ const TrackPage = ({
       following={following}
       onFollow={onFollow}
       onUnfollow={onUnfollow}
-      onDownload={onDownloadTrack}
```

Comment:
is download track already an action button on track page?

Reply:
yeah this was all greyed out because unused